_posts/2020-10-12-what-does-netways-do-with-foreman.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: What does Netways do with Foreman?
+date: 2020-10-12 10:08:27
+author: Dirk Götz
+tags:
+- foreman
+---
+
+As outlined in [Adding a vendor list to theforeman.org site](https://community.theforeman.org/t/adding-a-vendor-list-to-theforeman-org-site/20581), over the next few weeks, we will host some guest posts from different companies and consultants who offer professional services based on or extending Foreman for their customers. In this post, we hear from [@dirk](https://community.theforeman.org/u/dirk/summary) from [NETWAYS](https://www.netways.de/) and how they’ve interacted with the Foreman community to date. If you want to contribute to this series, please contact [@mcorr](https://community.theforeman.org/u/dirk/summary).
```

Comment:
The link for @mcorr is pointing to me. ;-)

Reply:
This is the best mistake I've ever made! 
Thank you :)